types/unit/timespan.pp

```diff
@@ -0,0 +1,4 @@
+# @summary Timer specification for systemd timers
+# @see https://www.freedesktop.org/software/systemd/man/systemd.timer.html
```

Comment:
```suggestion
# @see https://www.freedesktop.org/software/systemd/man/systemd.time.html
```
Sorry.

Reply:
Why sorry? I wish all puppet modules would have this kind of quality(-control) :smiley: